lib/client/Client.js

```diff
@@ -356,6 +356,32 @@ export default class Client {
   }
 
   /**
+   * Destroy multiple {@link Mailbox}es specified to the given _ids_ on the server.<br />
```

Comment:
IMO, I think that we should mention the fact that ids' order is important where we CANNOT delete a parent before deleting its parent.